target/scripts/build/packages.sh

```diff
@@ -97,6 +97,14 @@ function _install_packages
     "${MAIL_PROGRAMS_PACKAGES[@]}"
 }
 
+function _install_getmail
+{
+  GETMAIL_SID_DEB_URL=http://http.us.debian.org/debian/pool/main/g/getmail6/getmail6_6.18.9-1_all.deb
+  curl -Lkso getmail6.deb  "${GETMAIL_SID_DEB_URL}" && \
+  dpkg -i getmail6.deb 2>&1 && \
```

Comment:
```suggestion
  curl -Lkso getmail6.deb  "${GETMAIL_SID_DEB_URL}"
  dpkg -i getmail6.deb 2>&1
```

`&& \` are not needed, as we have `set -eE` at the top of the file.

Reply:
I've rebased against master again as I'd missed the latest ones with fail2ban in packages.sh and I've removed the && \ and made the variable "local" to match coding standard.